_posts/2021-01-23-how-we-co-operate.md

```diff
@@ -0,0 +1,174 @@
+
+  [interviews]: https://meetings.hypha.coop/#:~:text=Informational%20Interviews
+
+Our fellow co-operators, keepers of arcane regulatory knowledge,
+guided us through the incorporation process and development of bylaws.
+When we thanked them for their generosity, they’d shrug it off and say, “Principle 6.”
+Seeing the principle of “Co-operation Among Co-operatives” put into practice in everyday interactions was inspiring and reassured us that we were on the right course.
+It’s a phrase we love and have latched onto since.
+
+We’re grateful to [Fourth Pig][], [Anarres][] (RIP),
+[Root Systems][] (RIP), [Fiqus][], [Camba][], [FACTTIC][], [CWCF][],
+and many others who’ve guided us along the way.
+
+   [Fourth Pig]: https://www.fourthpig.org/
+   [Anarres]: http://web.archive.org/web/20160618102349/https://anarres.ca/
+   [Root Systems]: http://web.archive.org/web/20200122192017/https://www.rootsystems.nz/
+   [Fiqus]: https://fiqus.coop/en/
+   [Camba]: https://camba.coop/en/
+   [FACTTIC]: https://facttic.org.ar/
+   [CWCF]: https://canadianworker.coop/
+
+## Building the solidarity economy
+
+Participating in the co-op movement has been a respite from the broader individualizing and neoliberal society we’re trapped in.
+It has reaffirmed our belief that organizations who value solidarity,
+not private benefit, are more vital and resilient.
+
+Our experience here isn’t unique.
+Disenfranchised from capitalist economies,
+a growing number of tech workers are joining the co-operative movement.
+New models, like [platform co-operativism][],
+have emerged as alternatives to the dominant extractive forms of tech businesses.
+Tech workers are [taking collective action][],
+demanding better pay, equitable workplaces,
+and asking corporations to take responsibility for the social harms inflicted by technological practices. 
+
+   [platform co-operativism]: https://platform.coop/
+   [taking collective action]: https://logicmag.io/the-making-of-the-tech-worker-movement/full-text/
+
+This co-operative moment couldn’t have come sooner.
+As we rebuild our economy in the wake of COVID-19–and during this false calm that precedes reckoning with our looming climate crisis—we need interdependent,
+community-oriented, democratic, and ecologically-conscious organizations to take back our economy and ensure we have a [just recovery for all][].
+We’re excited to stand with others working to that end.
+
+   [just recovery for all]: https://justrecoveryforall.ca/
+
+This is the inaugural post of Hypha’s new blog, Dripline. Through Dripline, we’re hoping to share our experiences and reflections as we learn and build technologies together. It’s also an invitation to our friends and neighbours: [come help us build][] the solidarity economy. Principle 6.
+
+   [come help us build]: https://www.eyemole.io/startacoop.html
+
+We’re excited to hear from you!
+Reach out if there are topics you want to discuss or you think we should cover in future posts.
```

Comment:
I would add this in the blog page footer for all dripline posts. This isn't specific to this post.